src/app/containers/Video/videoMetadata.js

```diff
@@ -26,6 +26,7 @@ const videoMetadata = blocks => {
       description: deepGet(['model', 'synopses', 'short'], block),
       duration: deepGet(['model', 'versions', [0], 'duration'], block),
       thumbnailUrl: `https://${deepGet(['model', 'imageUrl'], block)}`,
+      uploadDate: deepGet(['model', 'versions', [0], 'availableUntil'], block),
```

Comment:
The field used in the JSON should be `availableFrom` - as was mentioned in the original issue. 

```suggestion
      uploadDate: deepGet(['model', 'versions', [0], 'availableFrom'], block),
```
An example video that has this field, for use in unit tests, can be this one: https://www.test.bbc.co.uk/news/articles/c4d8dqrd5wyo.json